### PR TITLE
update English document

### PR DESCRIPTION
**Pull request by DogHeadMoon**
update English document, including

piecewise decay
natural exponential decay
exponential decay
inverse time decay
polynomial decay
cosine decay
noam decay

Conv3D
Conv3DTranspose
GRUUnit
BilienarTensorProduct

label_smoothing

### CODE REVIEW

**Comment by DogHeadMoon** [comment]
Piecewise Decay
image

**Comment by DogHeadMoon** [comment]
natural exponential decay
image

**Comment by DogHeadMoon** [comment]
exponential decay
image

**Comment by DogHeadMoon** [comment]
inverse time decay
image

**Comment by DogHeadMoon** [comment]
polynomial decay
image

**Comment by DogHeadMoon** [comment]
cosine decay 
image

**Comment by DogHeadMoon** [comment]
noam decay
image

**Comment by DogHeadMoon** [comment]
Conv3d
image

**Comment by DogHeadMoon** [comment]
Conv3dTranspose
image

**Comment by DogHeadMoon** [comment]
GRUUnit
image

**Comment by DogHeadMoon** [comment]
Bilinear
image

**Comment by DogHeadMoon** [comment]
label_smoothing
image

File: python/paddle/fluid/dygraph/learning_rate_scheduler.py [hunk 1]
```diff
@@ -69,30 +69,33 @@ def step(self):
...
+
+    Parameters:
+        boundaries(list): A list of steps numbers. The type of element in the list is python int.
+        values(list): A list of learning rate values that will be picked during
+            different step boundaries. The type of element in the list is python float.
+        begin(int): The begin step to initilize the global_step in the description above.
+        step(int, optional): The increasementation used to calculate the new global_step in the description above.
+            The defalult value is 1.
+        dtype(str, optional): The data type used to create the learning rate variable. The data type can be set as
+            float32, float64. The default value is 'float32'.
```
Comment (phlrain) [inline python/paddle/fluid/dygraph/learning_rate_scheduler.py]: 'float32', 'float64'

File: python/paddle/fluid/dygraph/learning_rate_scheduler.py [hunk 2]
```diff
@@ -125,25 +128,40 @@ class NaturalExpDecay(LearningRateDecay):
...
+            float32 or float64. It also can be set to python int number.
+        decay_steps(int): The decay step size. It determines the decay cycle.
+        decay_rate(int): The decay rate.
+        staircase(bool, optional): If set to True, decay the learning rate at discrete intervals. The 
+            default value is False.
+        begin(int, optional): The begin step. The initial value of global_step described above. The default value is 0.
+        step(int, optional): The increasementation used to calculate the new global_step in the description above.
+            The defalult value is 1.
+        dtype(str, optional): The data type used to create the learning rate variable. The data type can be set as
+            float32, float64. The default value is 'float32'.
```
Comment (phlrain) [inline python/paddle/fluid/dygraph/learning_rate_scheduler.py]: 同上

File: python/paddle/fluid/dygraph/learning_rate_scheduler.py [hunk 3]
```diff
@@ -189,29 +207,41 @@ class ExponentialDecay(LearningRateDecay):
...
+            float32 or float64. It also can be set to python int number.
+        decay_steps(int): The decay step size. It determines the decay cycle.
+        decay_rate(float): The decay rate.
+        staircase(bool, optional): If set to True, decay the learning rate at discrete intervals. The 
+            default value is False.
+        begin(int, optional): The begin step. The initial value of global_step described above. The default value is 0.
+        step(int, optional): The increasementation used to calculate the new global_step in the description above.
+            The defalult value is 1.
+        dtype(str, optional): The data type used to create the learning rate variable. The data type can be set as
+            float32, float64. The default value is 'float32'.
```
Comment (phlrain) [inline python/paddle/fluid/dygraph/learning_rate_scheduler.py]: 同上

File: python/paddle/fluid/dygraph/learning_rate_scheduler.py [hunk 4]
```diff
@@ -257,27 +287,35 @@ class InverseTimeDecay(LearningRateDecay):
...
+            float32 or float64. It also can be set to python int number.
+        decay_steps(int): The decay step size. It determines the decay cycle.
+        decay_rate(float): The decay rate.
+        staircase(bool, optional): If set to True, decay the learning rate at discrete intervals. The 
+            default value is False.
+        begin(int, optional): The begin step. The initial value of global_step described above. The default value is 0.
+        step(int, optional): The increasementation used to calculate the new global_step in the description above.
+            The defalult value is 1.
+        dtype(str, optional): The data type used to create the learning rate variable. The data type can be 
+            float32, float64. The default value is 'float32'.
```
Comment (phlrain) [inline python/paddle/fluid/dygraph/learning_rate_scheduler.py]: 同上

File: python/paddle/fluid/dygraph/learning_rate_scheduler.py [hunk 5]
```diff
@@ -257,27 +287,35 @@ class InverseTimeDecay(LearningRateDecay):
...
+    Parameters:
+        learning_rate(Variable|float): The initial learning rate. If the type 
+            is Variable, it's a tensor with shape [1], the data type can be  
+            float32 or float64. It also can be set to python int number.
+        decay_steps(int): The decay step size. It determines the decay cycle.
+        decay_rate(float): The decay rate.
+        staircase(bool, optional): If set to True, decay the learning rate at discrete intervals. The 
+            default value is False.
+        begin(int, optional): The begin step. The initial value of global_step described above. The default value is 0.
+        step(int, optional): The increasementation used to calculate the new global_step in the description above.
```
Comment (phlrain) [inline python/paddle/fluid/dygraph/learning_rate_scheduler.py]: increasementation 这个单词太复杂了。

File: python/paddle/fluid/dygraph/learning_rate_scheduler.py (hunk 1 above)
Comment (DogHeadMoon) [reply python/paddle/fluid/dygraph/learning_rate_scheduler.py]: 已更正

File: python/paddle/fluid/dygraph/learning_rate_scheduler.py (hunk 2 above)
Comment (DogHeadMoon) [reply python/paddle/fluid/dygraph/learning_rate_scheduler.py]: 已更正

File: python/paddle/fluid/dygraph/learning_rate_scheduler.py (hunk 3 above)
Comment (DogHeadMoon) [reply python/paddle/fluid/dygraph/learning_rate_scheduler.py]: 已更正

File: python/paddle/fluid/dygraph/learning_rate_scheduler.py (hunk 4 above)
Comment (DogHeadMoon) [reply python/paddle/fluid/dygraph/learning_rate_scheduler.py]: 已更正

File: python/paddle/fluid/dygraph/learning_rate_scheduler.py (hunk 5 above)
Comment (DogHeadMoon) [reply python/paddle/fluid/dygraph/learning_rate_scheduler.py]: 已替换为step size

**Review by phlrain — DISMISSED** [review]

**Review by phlrain — DISMISSED** [review]

**Review by wanghaoshuang — DISMISSED** [review]

**Review by kuke — DISMISSED** [review]

**Review by phlrain — APPROVED** [review]

**Review by wanghaoshuang — APPROVED** [review]

**Review by heavengate — APPROVED** [review]

**Pull request merged**